### ISSUE TITLE
Partitioning Logic

### ISSUE THREAD

**Issue by outbackdingo** [OverC/meta-overc#297]
Can someone give me a briefing on the partitioning logic for OverC ? basically break down the partitions and whats used for what ?

**Comment by zeddii** [comment]
by partitioning, do you mean what services run in what system container ? There's an overview in the READMEs.

**Comment by outbackdingo** [comment]
No, I mean physical disk partitions / slices..... as you create quite a few....
